log4j-audit/log4j-audit-api/src/test/java/org/apache/logging/log4j/audit/LogoutTest.java

```diff
@@ -16,27 +16,27 @@
  */
 package org.apache.logging.log4j.audit;
 
-import org.apache.logging.log4j.audit.event.Login;
+import org.apache.logging.log4j.audit.event.Logout;
```

Comment:
I've renamed this event and `Login` was already defined in the JSON catalog, so an interface with the same name and package was also generated from the catalog and I wanted to make sure they don't collide.